### DISCUSSION TITLE
Object 365 Checkpoints

### DISCUSSION THREAD

**Discussion by Jacobsolawetz** [ultralytics/ultralytics#1962] [Ideas]
Have Objects 365 checkpoints been release? We are planning to do some work on how yolo evals training from those vs COCO across Roboflow 100. 

If not, we are happy to contribute them!

**Comment by Jacobsolawetz** [comment]
Also wondering - when you are training YOLOv8 on multi-gpu, which folder do you expect the weights to get saved? In this case, I took the ones from `objects-365-pretraining-s9` - does that seem reasonable?
![image](https://user-images.githubusercontent.com/8428198/231178876-7aec14b7-cdab-4140-914f-9b5e886f8c38.png)

**Reply by glenn-jocher** [reply]
@Jacobsolawetz hey buddy. No, we don't have Objects365 pretrained models other than YOLOv5m in a prior release, perhaps v6.0. 

Both DDP and Single-GPU create only 1 runs directory where all results are saved to. This is indicated at the start and end of training, or you can define your own runs directory as PROJECT/NAME, i.e.:

```python
yolo train project=PROJECT name=NAME
```

**Reply by glenn-jocher** [reply]
For DDP BTW the command is pretty simple, i.e.:

```python
# Train Single-GPU
yolo train device=0

# Train Multi-GPU
yolo train device=0,1,2,3,4,5,6,7
```

**Reply by glenn-jocher** [reply]
For details and a full list of training args see https://docs.ultralytics.com/modes/train/
